src/main/scala/uk/gov/nationalarchives/fileformat/LambdaRunner.scala

```diff
@@ -0,0 +1,17 @@
+package uk.gov.nationalarchives.fileformat
+
+import com.amazonaws.services.lambda.runtime.events.SQSEvent
+import com.amazonaws.services.lambda.runtime.events.SQSEvent.SQSMessage
```

Comment:
Not sure this import is needed? `import com.amazonaws.services.lambda.runtime.events.SQSEvent.SQSMessage`

Reply:
It's not needed and the entire library isn't needed. I've removed this and the AWS library they're imported from.